### DISCUSSION TITLE
howto revert/undo microcode upgrade?          io wait issue in guests

### DISCUSSION THREAD

**Discussion by AuZ-ASD** [community-scripts/ProxmoxVE#592] [Q&A]
RIP tteck!

Hi All,
I thought i would be a good Idea to update the microcode of my intel N100 and I did so with the help of "https://tteck.github.io/Proxmox/#proxmox-ve-processor-microcode" the update offered two different versions and i picket the newer one

>  ( ) intel-microcode_3.20240910.1~deb12u1_amd64.deb
>  (*) intel-microcode_3.20241112.1_amd64.deb  

all went smooth, I rebootet and confirmed i have now some

>  microcode: Current revision: 0x0000001a

_I also changed the powersaving stuff at the same time...(https://tteck.github.io/Proxmox/#proxmox-ve-cpu-scaling-governor)

yea i know... never change two things at the same time... sorry i did...  but the powersaving thing is easy to revert ... just set it back to "performance" which i already did._

but i Noticed some lag within my VMs... later i found: since the update my IO wait within the guests is like hell! 800ms and more!
[diskstat_latency_sda-week](https://github.com/user-attachments/assets/09f91353-1b8b-4d5c-a563-14586cbc668b)
as already mentioned the powersaving change was already reverted but it is still bad.

side note... my cluster is 3times the same hardware all 3 nodes have goten the same microcode etc... but it looks like only on node has this high io wait on the disk ceth uses... the other two nodes have normal times there....
SMART values dont show any bad sigh for this disk...

**so now the question: how do i undo/revert this microcode update?**

proxmox:   8.3.0          (updated & rebooted on 26.11. and after this the io wait issue was NOT there)
ceth:  18.2.4            (comes with proxmox 8.3)

**Comment by tremor021** [comment]
https://metadata.ftp-master.debian.org/changelogs/non-free/i/intel-microcode/unstable_README.Debian
